json_serializable/test/test_files/kitchen_sink.nullable.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// GENERATED CODE - DO NOT MODIFY BY HAND
+
+// **************************************************************************
+// Generator: Test Boilerplate generator
+// **************************************************************************
+
+// ignore_for_file: annotate_overrides, hash_and_equals
+library json_serializable.test.kitchen_sink_nullable;
+
+import 'package:json_annotation/json_annotation.dart';
+
+import 'kitchen_sink.dart';
+import 'test_files_util.dart';
+
+part 'kitchen_sink.nullable.g.dart';
+
+List<T> _defaultList<T>() => <T>[];
+Map _defaultMap() => {};
+
+@JsonSerializable(nullable: false)
+class KitchenSinkNullable extends Object
```

Comment:
Isn't this really a `KitchenSinkNonNullable`?

Reply:
Yup. PTAL